lib/unicore/mktables

```diff
@@ -10824,7 +10824,7 @@ sub output_perl_charnames_line ($a, $b) {
     # ordinals, but they are all private use or noncharacters which do not
     # have names, so won't be in this table.
 
-    return sprintf "%05X\n%s\n\n", $_[0], $_[1];
+    return sprintf "%05X\n%s\n\n", $a, $b;
```

Comment:
I know this is existing code, but while we're here, please rename these to something descriptive, like `$codepoint` and `$name`.

Reply:
I'll do that in a separate commit